runtime/lua/vim/_meta/vimfn.lua

```diff
@@ -1751,6 +1751,12 @@ function vim.fn.exists(expr) end
 --- @return any
 function vim.fn.exp(expr) end
 
+--- @param string string
+--- @param nosuf? boolean
+--- @param list? nil|false
+--- @return string
+function vim.fn.expand(string, nosuf, list) end
```

Comment:
overloaded functions should come after the docs. so can we move this to afters after L1854? (or swap the order if appropriate)

Reply:
or another way is to use `@overload` https://luals.github.io/wiki/annotations/#overload

Reply:
From my experience I haven't found `@overload` to work well.